posts/physical-backup-restore.md

```diff
@@ -0,0 +1,457 @@
+---
+title: "Physical Backup & Restoration"
+description: "Physical Backup & Restoration"
+date: "2025-04-09"
+is_redirect: false
+redirect_url:
+---
+
+
+In Frappe Cloud, every **Site Update** triggers an automatic backup - ensuring your data stays safe in case of failures and we can rollback your site to older version as soon as possible.
+
+As of now, we use `mysqldump` to take logical backup your site's database.
+
+```
+mysqldump \
+  --single-transaction --quick \
+  --lock-tables=false \
+  -u root -psecret \
+  _c678923hj223 \
+  > backup.sql
+```
+
+This works fine !
```

Comment:
```suggestion
This works fine.
```

Use `!` sparingly. In this case you also want to continue the sentence to setup the section. E.g. "This works fine for the most part but it doesn't scale well with growing database sizes.